src/docs/firmware/supported-devices.md

```diff
@@ -2231,4 +2231,5 @@ If on on a smaller screen, scroll sideways to see whole table.
         </tr>
     </tbody>
 </table>
+
```

Comment:
?

Reply:
I think it was an accident that came up when trying to fix merge conflicts earlier